proto/decentraland/realm/about.proto

```diff
@@ -10,10 +13,55 @@ message AboutResponse {
   optional BffInfo bff = 6;
   bool accepting_users = 7;
 
-  message MinimapConfiguration {
-    bool enabled = 1;
-    optional string data_image = 2;
-    optional string estate_image = 3;
+  message MapConfiguration {
+    // wheter the minimap could be rendered or not
+    bool minimap_enabled = 1;
+  
+    // the union of all rects here represents the places where that MAY have scenes
+    //  - all the other parcels are considered as empty, up to the explorer to decide if they're walkable or not
+    //  - the contained square is determined by points: 
+    //    - top-left      with  minX, maxY 
+    //    - bottom-left   with  minX, minX 
```

Comment:
```suggestion
    //    - bottom-left   with  minX, minY
```

I think the review was not applied correctly

Reply:
thanks